charts/alloy-api/values.yaml

```diff
@@ -0,0 +1,153 @@
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+# If this deployment needs to trust non-public certificates,
+# - Enable this option
+# - Place the certificates in a Volume and create a PV and PVC for them
+# - Enter the name for the PVC as pvcName
+applyCertificates:
+  enabled: false
+  pvcName: ""
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+command: [ "bash", "-c", "update-ca-certificates && dotnet Alloy.Api.dll" ]
+
+# env is pod env vars
+env:
+  http_proxy: 
+  https_proxy:
+  HTTP_PROXY:
+  HTTPS_PROXY: 
+  NO_PROXY: 
+  no_proxy: 
+
+  Logging__IncludeScopes: false
+  Logging__Debug__LogLevel__Default: Information
+  Logging__Debug__LogLevel__Microsoft: Error
+  Logging__Debug__LogLevel__System: Error
+  Logging__Console__LogLevel__Default: Information
+  Logging__Console__LogLevel__Microsoft: Error
+  Logging__Console__LogLevel__System: Error
+
+  ConnectionStrings__PostgreSQL: "<connection string>"
+  Database__AutoMigrate: true
+  Database__DevModeRecreate: false
+  Database__Provider: PostgreSQL
+
+  CorsPolicy__Origins__0: http://localhost:4403
+  CorsPolicy__Methods__0: ""
+  CorsPolicy__Headers__0: ""
+  CorsPolicy__AllowAnyOrigin: false
+  CorsPolicy__AllowAnyMethod: true
+  CorsPolicy__AllowAnyHeader: true
+  CorsPolicy__SupportsCredentials: true
+
+  Authorization__Authority: ""
+  Authorization__AuthorizationUrl: ""
+  Authorization__AuthorizationScope: "alloy-api player-api caster-api steamfitter-api vm-api"
```

Comment:
I wonder since your apps are fairly tightly integrated in their own ecosystem, if a single `crucible-api` could be honored by all of them?